requirements/base.txt

```diff
@@ -1,2 +1,9 @@
-# Base requirements are empty. All of our minimum requirements are specified in setup.py's install_requires variable!
--e .
+django-filter==0.13.0
+django-js-reverse==0.7.2
+six==1.10.0
+djangorestframework==3.3.3
+django==1.9.7
+colorlog==2.6.0
+docopt==0.6.2
+django-mptt==0.8.5
+https://github.com/cherrypy/cherrypy/archive/v6.2.0.zip
```

Comment:
Cherrypy's source code is designed to work for both python 2 and python 3, unmodified. Unfortunately it seems like downloading it from PyPI adds in a restriction such that a separate sdist needs to be downloaded for both versions of python, i.e. when you download an sdist while using python 2, that sdist will not be installable for python 3. Kolibri inherits this restriction, and unless we somehow find a way to have CherryPy work on both python 2 and 3, we need separate sdists for both versions.